src/FastBoard.cpp

```diff
@@ -38,19 +38,22 @@ const int FastBoard::BIG;
 const int FastBoard::PASS;
 const int FastBoard::RESIGN;
 
+/* indicates that an empty square is never in atari. A large value that fits in a short */
+const int MAX_LIBS = 16384;  // 2^14
```

Comment:
Move the definition to the header for consistency. It should be a constexpr as well. Also it's not really a max value. INF_LIBS?

Reply:
Moved it to .h and renamed it to INF_LIBS.